lib/NormalModuleFactory.js

```diff
@@ -641,7 +649,12 @@ class NormalModuleFactory extends ModuleFactory {
 			}
 
 			if (typeof result === "object")
-				throw new Error(deprecationChangedHookMessage("beforeResolve"));
+				throw new Error(
+					deprecationChangedHookMessage(
+						"beforeResolve",
+						this.hooks.beforeResolve.tap.name
```

Comment:
actually the name comes from `this.hooks.hookName.taps[].name`, what is the best way to get the name here?

Reply:
I choose to go with taps, is there a better way anyways?